source/language/types.rst

```diff
@@ -583,6 +641,21 @@ Casting specifics
 The classical types are divided into the 'standard' classical types (bool, int, 
 uint, and float) that exist in languages like C, and the 'special' classical 
 types (bit, angle, duration, and stretch) that do not.
+The standard types follow rules that mimick those of C99 for `promotion and
```

Comment:
```suggestion
The standard types follow rules that mimic those of C99 for `promotion and
```

Reply:
Changed in https://github.com/Qiskit/openqasm/pull/337/commits/e6e1c2835d4b57e656f9fc141f3a9187db210930